crontab

```diff
@@ -1,6 +1,6 @@
 # Run weekly on Wednesday at Noon PDT
-# 0 19 * * WED java -Dserver.port=9191 -jar /opt/la-doc-uploader/app.jar transmit
+# 0 19 * * WED java -Dserver.port=9191 -jar /opt/form-flow-starter-app/app.jar transmit
 
 # Run every hour
-# @hourly java -Dserver.port=9191 -jar /opt/la-doc-uploader/app.jar transferSubmissions
+@hourly java -Dserver.port=9191 -jar /opt/form-flow-starter-app/app.jar transferSubmissions
```

Comment:
@sree-cfa - curious: why is this referencing the form-flow starter app instead of the la-doc-uploader?

Reply:
This is just the path we've been using in the Dockerfile - the "MKDIR" part